ibis/bigquery/tests/conftest.py

```diff
@@ -0,0 +1,54 @@
+import pytest
+
+import pandas as pd
+import ibis.util
+bq = pytest.importorskip('google.datalab.bigquery')
+from ibis.bigquery import client as ibc  # noqa: E402
+
+
+PROJECT_ID = 'bq-for-ibis'
+DATASET_ID = 'ci_{}'.format(ibis.util.guid())
+TABLE = 'table_{}'.format(ibis.util.guid())
+
+
+@pytest.fixture(scope='session')
+def df():
+    return pd.DataFrame(dict(
+        int_column=[1], float_column=[2.],
+        string_column=['a'], bool_column=[True],
+    ))
+
+
+@pytest.fixture(scope='session')
+def client():
+    return ibc.BigQueryClient(PROJECT_ID, DATASET_ID)
+
+
+@pytest.fixture(scope='session')
```

Comment:
This needs to be `yield_fixture`.

Reply:
For consistency or correctness?

https://docs.pytest.org/en/latest/yieldfixture.html

> Since pytest-3.0, fixtures using the normal fixture decorator can use a yield statement to provide fixture values and execute teardown code, exactly like yield_fixture in previous versions.
> 
> Marking functions as yield_fixture is still supported, but deprecated and should not be used in new code.

Reply:
Oh sweet! Thanks for the link.